woocommerce/woocommerce-gutenberg-products-block
# Add: Interactivity Active Filters block

## What

This PR adds the new Active Filters Block powered by Interactivity API. It has the same features as the current one and works with new filter blocks.

Fixes woocommerce/woocommerce#42179

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add the `Collection Filters` block into a `Product Collection` block.
2. See the `Collection Active Filters` and other filter blocks added along with the `Collection Filters` block.
3. Toggle the display style, see the editor reacts.
4. Save and test the block on the frontend:
  - It's hidden when no filter applied.
  - It's visible when a filter is applied.
  - Clicking on the remove button of each active filter removes that filter from the URL and update the products grid as well as the corresponding filter block.
  - Clicking on the `Clear All` button remove all filters.
  - Removing a filter from the filter block also remove the corresponding one from the active filter block.
  - See chip and list style work as expected.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
